src/views/Home/components/MarketData.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+
+import numeral from 'numeral'
+import { Container, Card, CardContent, Spacer } from 'react-neu'
+
+import FancyValue from 'components/FancyValue'
+import Split from 'components/Split'
+import SimplePriceChart from 'components/SimplePriceChart'
+
+import useDpiTokenMarketData from 'hooks/useDpiTokenMarketData';
+
+const MarketData: React.FC = () => {
+  const {
+    latestVolume,
+    latestMarketCap,
+    latestPrice,
+    prices
+  } = useDpiTokenMarketData()
+  const priceAtEpochStart = (prices || [[0, 1]])[0][1]
```

Comment:
The second part of this expression is sort of hard to understand.
Our repo has access to [optional chaining which is pretty cool](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

It looks sort of weird with dynamic array indices but it would work like this here:
`const priceAtEpochStart = (prices?.[0]?.[1] || 1)`

Reply:
ahh ok I was trying that, didn't realize you had to add the . after ? so I figured it wasn't possible with arrays. Thanks.